app/jobs/sample_manifest/generate_wells_job.rb

```diff
@@ -4,7 +4,7 @@
   Struct.new(:sample_manifest_id, :map_ids_to_sanger_sample_ids, :plate_id) do
     def perform
       ActiveRecord::Base.transaction do
-        map_ids_to_sanger_sample_ids.each { |k, v| create_well(k, v) }
+        map_ids_to_sanger_sample_ids.each { |map_id, sanger_sample_id| create_well(map_id, sanger_sample_id) }
```

Comment:
Perhaps the work around for this variable renaming pain is to use that Ruby syntax where you define a method to pass the values to instead of write it out in full like you have here.  I don't know if this works when you have two values coming in, but I know you can push the values through another method when you write it something like:

```suggestion
        map_ids_to_sanger_sample_ids.each &:create_well
```